conjurapi/api_key_authenticator.go

```diff
@@ -1,47 +1,17 @@
 package conjurapi
 
-import (
-	"strings"
-	"io/ioutil"
-	"fmt"
-	"net/http"
-	"time"
-	"net/url"
-)
-
 type APIKeyAuthenticator struct {
-	AuthnURLTemplate string
-	Login string `env:"CONJUR_AUTHN_LOGIN"`
-	APIKey string `env:"CONJUR_AUTHN_API_KEY"`
+	Authenticate func (loginPair LoginPair) ([]byte, error)
```

Comment:
Whenever we export (uppercase) a go type or var to a package we should godoc it too. We can do this as a followup task after all these PRs are merged.